HolyShift.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
Usually we have an *.sln in the root directly. With a folder for each *.csproj

Are you using rider to generate the project or what are you doing?